ros2_ws/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.5)
+project(pointcloud2_pubsub)
+
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(sensor_msgs REQUIRED)
+
+add_executable(publisher_node src/publisher_node.cpp)
+ament_target_dependencies(publisher_node rclcpp sensor_msgs)
+
+add_executable(subscriber_node src/subscriber_node.cpp)
+ament_target_dependencies(subscriber_node rclcpp sensor_msgs)
+
+install(TARGETS
+  publisher_node
+  subscriber_node
+  DESTINATION lib/${PROJECT_NAME}
+)
+
+ament_package()
```

Comment:
Although `ament_package()` is correctly used,  consider adding specific package information such as description, maintainers, and dependencies within the `ament_package()` call for better documentation and package management. This is particularly important for larger projects and collaboration.